python_modules/dagster/dagster/_core/execution/context/invocation.py

```diff
@@ -671,6 +712,7 @@ def build_op_context(
     instance: Optional[DagsterInstance] = None,
     config: Any = None,
     partition_key: Optional[str] = None,
+    partition_key_range: Optional[PartitionKeyRange] = None,
```

Comment:
i just landed `build_asset_context` so will need to update that to support this argument as well

Reply:
done